### PR TITLE
Feature/cookie auth

### PR DESCRIPTION
**Pull request by IvaskevychYuriy**

### CODE REVIEW

File: SimpleDrive.App/Controllers/HomeController.cs [hunk 1]
```diff
@@ -1,19 +1,104 @@
...
+                if (result.Succeeded)
+                {
+                    return StatusCode(code, await MapToProfile(user));
+                }
+            }
+
+            return Unauthorized();
+        }
+
+        private async Task<UserProfileDTO> MapToProfile(User user)
```
Comment (nazar554) [inline SimpleDrive.App/Controllers/HomeController.cs]: DTO is cool but why? we need claims

File: SimpleDrive.App/Extensions/WebHostExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        public static void SeedData(this IWebHost host)
+        {
+            var scopeFactory = host.Services.GetRequiredService<IServiceScopeFactory>();
+            using (var scope = scopeFactory.CreateScope())
+            {
+                var db = scope.ServiceProvider.GetService<ApplicationDbContext>();
+
+                var roleManager = scope.ServiceProvider.GetRequiredService<RoleManager<Role>>();
+                var adminRole = roleManager.FindByNameAsync("Administrator").Result;
```
Comment (nazar554) [inline SimpleDrive.App/Extensions/WebHostExtensions.cs]: make constants or move to config: `"Administrator"`, `"User`

**Review by nazar554 — CHANGES_REQUESTED** [review]
check if we really need DTOs for User
if we do use AutoMapper

File: SimpleDrive.App/Controllers/HomeController.cs (hunk 1 above)
Comment (IvaskevychYuriy) [reply SimpleDrive.App/Controllers/HomeController.cs]: Why do we need claims here?

Comment (IvaskevychYuriy) [reply SimpleDrive.App/Controllers/HomeController.cs]: @nazar554 Fixed in 54f7112. Propose to add claims later on when we have more vision on that.

File: SimpleDrive.App/Extensions/WebHostExtensions.cs (hunk 2 above)
Comment (IvaskevychYuriy) [reply SimpleDrive.App/Extensions/WebHostExtensions.cs]: @nazar554 Fixed in 54f7112.

**Review by nazar554 — APPROVED** [review]

**Pull request merged**